source/code/plugin/lib/application_insights/channel/asynchronous_sender.rb

```diff
@@ -20,15 +20,16 @@ class AsynchronousSender < SenderBase
       SERVICE_ENDPOINT_URI = 'https://dc.services.visualstudio.com/v2/track'
       # Initializes a new instance of the class.
       # @param [String] service_endpoint_uri the address of the service to send
+      # @param [Hash] proxy server configuration to send (optional)
```

Comment:
This should go to AI Ruby SDK and not checked in as part of our code. We just pull the gem and use it.

Reply:
Done. Same change made to AI SDK repo as well, but yet to merged that PR.